### DISCUSSION TITLE
Nether

### DISCUSSION THREAD

**Discussion by mataaj** [pmdevita/StayPut#30] [Q&A] [answered]
Is there a way to save players last location in nether? The plugin unfortunately doesnt seem to work there, not sure if thats an issue on my side or not.

**Comment by pmdevita** [comment, marked as answer]
It should be saving your location in the nether, turn on debug mode and send me your server logs when you teleport in and out of the nether

**Reply by mataaj** [reply]
I could still send the log if you want but I´m pretty sure I now know whats the issue. My lobby world has the default name "world" but the world connected to the nether is called "Survival" causing the issue. So my last location actually does save, it just doesnt recognise the link between those worlds since the plugin doesnt support world groups, right?

**Reply by pmdevita** [reply]
Ah that might be the case, yeah the nether needs to be named survival_nether to link it with survival. This will be configurable in the future but right now the plugin does it automatically.
